grpc/src/main/java/com/linecorp/armeria/server/grpc/UnaryServerCall.java

```diff
@@ -70,11 +70,12 @@ final class UnaryServerCall<I, O> extends AbstractServerCall<I, O> {
                     ServiceRequestContext ctx, SerializationFormat serializationFormat,
                     @Nullable GrpcJsonMarshaller jsonMarshaller, boolean unsafeWrapRequestBuffers,
                     ResponseHeaders defaultHeaders,
-                    @Nullable GrpcStatusFunction statusFunction, @Nullable Executor blockingExecutor,
+                    @Nullable GrpcExceptionHandlerFunction grpcExceptionHandlerFunction,
```

Comment:
nit: For simplicity.
```suggestion
                    @Nullable GrpcExceptionHandlerFunction exceptionHandler,
```